resources/js/Pages/Login.vue

```diff
@@ -3,6 +3,12 @@
     <Head title="Login" />
     <Menu />
     <section>
+        <b-message
+                    title="ERROR"
+                    type="is-danger"
+                    aria-close-label="Close message" v-if="error">
+                        {{ error }}
+        </b-message>
         <div class="box">
             <form action="/loginCheck" method="POST" @submit.prevent="login">
```

Comment:
- I'm nitpicking but because we use Inertia, the `action` tag in the form tag (in HTML) aren't needed.